packages/browser-agent-core/common/window/nreum.js

```diff
@@ -94,6 +94,26 @@ export function gosNREUMInitializedAgents(id, obj, target) {
   return nr
 }
 
+function gosNRDEBUG() {
```

Comment:
Previously this code was injected by JIL when tests were running. Is this temporary/will this be added to the production agent?

Reply:
After thinking, I think this should actually be temporary

Reply:
> After thinking, I think this should actually be temporary (and cleaned up)

Done in the cleanup branch; disregard this on here for now. Alternatively, can merge that in but not preferable as it'll bloat this one and make review harder for everyone to go through a bunch of commented-out removals.